src/mudlet.cpp

```diff
@@ -934,7 +944,7 @@ void mudlet::scanForMudletTranslations(const QString& path)
                 }
             }
             // PLACEMARKER: Start of locale codes to native language decoding - insert an entry here for any futher Mudlet supported languages
-            translation currentTranslation(percentageTranslated);
+            translation currentTranslation(percentageTranslated, languageCode);
```

Comment:
This seems wrong and I am not sure I like this - it is duplicating information because the `languageCode` is what is used as the **key** when it is stored in the `(QMap<QString, translation>) mudlet::mTranslationsMap` storing it as well in the **value** part of that **QMap** is unneeded.

Reply:
I think you're right, I'll look into revising this.

Reply:
@SlySven I've fixed it.